app/assets/stylesheets/views/_landing_page/main-navigation.scss

```diff
@@ -0,0 +1,30 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-main-navigation {
+  padding-top: govuk-spacing(3);
```

Comment:
Small nitpick, you could use shorthand here for all the padding values, either old style with four values, or new style using `padding-block` and `padding-inline`. But happy to approve it as is if you're doing it this way for clearer readability. Same comment for margins at lines 18-19.

Reply:
Thanks Matt - yep I find being more verbose more readable, but can combine them into one line if preferred.

Reply:
No worries, happy to leave these as is.